app/graphql/types/namespace_type.rb

```diff
@@ -31,6 +35,10 @@ class NamespaceType < Types::BaseObject
 
     id_field Namespace
     timestamps
+
+    def project(id:)
+      object.projects.find_by(id: id)
```

Comment:
`id` is a `GlobalID` object here and not the numeric id that we need to the SQL query.

```suggestion
      object.projects.find_by(id: id.model_id)
```